packages/commons/src/events/ActionConfig.ts

```diff
@@ -149,19 +150,26 @@ export type InferredActionConfig =
   | z.infer<typeof RejectCorrectionConfig>
   | z.infer<typeof ApproveCorrectionConfig>
 
-export const ActionConfig = z.discriminatedUnion('type', [
-  DeclareConfig,
-  ValidateConfig,
-  RejectDeclarationConfig,
-  MarkedAsDuplicateConfig,
-  ArchiveConfig,
-  RegisterConfig,
-  DeleteConfig,
-  PrintCertificateActionConfig,
-  RequestCorrectionConfig,
-  RejectCorrectionConfig,
-  ApproveCorrectionConfig
-]) as unknown as z.ZodDiscriminatedUnion<'type', AllActionConfigFields[]>
+export const ActionConfig = z
+  .discriminatedUnion('type', [
+    DeclareConfig.openapi({ ref: 'DeclareActionConfig' }),
+    ValidateConfig.openapi({ ref: 'ValidateActionConfig' }),
+    RejectDeclarationConfig.openapi({ ref: 'RejectDeclarationActionConfig' }),
+    MarkedAsDuplicateConfig.openapi({ ref: 'MarkedAsDuplicateActionConfig' }),
+    ArchiveConfig.openapi({ ref: 'ArchiveActionConfig' }),
+    RegisterConfig.openapi({ ref: 'RegisterActionConfig' }),
```

Comment:
@makelicious thoughts? These need to be added if we want the models to show properly in OpenAPI
![image](https://github.com/user-attachments/assets/1ed4a922-a313-432a-b35d-6231782b0569)